src/act_cell.py

```diff
@@ -38,14 +38,19 @@ def state_size(self):
 
     def __call__(self, inputs, state, timestep=0, scope=None):
 
+        self.state_is_tuple = False
```

Comment:
You can do something like: 
```
if hasattr(self.cell, "_state_is_tuple"):
    self._state_is_tuple = self.cell._state_is_tuple
else:
    self._state_is_tuple = False
```
This way, it could be in the constructor, rather than being done every time the cell is called.